packages/test/mocha-test-setup/api-report/mocha-test-setup.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-/// <reference types="mocha" />
```

Comment:
The tooling likes to introduce this change intermittently (known bug). We should probably revert this here to reduce file churn.